### DISCUSSION TITLE
CLI tasks

### DISCUSSION THREAD

**Discussion by IKaramitsos** [upatras-hci/iv#154] [Q&A] [answered]
Κατα την ανάγνωση των αναλυτικών οδηγιών, δεν έχω καταλάβει ποιο πρέπει να είναι το παραδοτέο για την 8η εβδομάδα.

Συγκεκριμένα αυτό που κατάλαβα είναι η επιλογή ασκήσεων από την "[Λίστα διαθέσιμων ασκήσεων γραμμής εντολών](https://github.com/epidrome/dokey)" και η οπτικοποίηση τους σε γραμμή εντολών ανάλογα με τα δεδομένα που είναι σημαντικά για εμένα, από οποιαδήποτε κατηγορία : 

[warm up](https://github.com/epidrome/dokey#warmup): basic skills and gui alternatives
[hci](https://github.com/epidrome/dokey#hci): user land
[cscw](https://github.com/epidrome/dokey#cscw): collaborate with users and devices
[visualization](https://github.com/epidrome/dokey#visualization): rich content
[software](https://github.com/epidrome/dokey#software): dev ops

Έχω καταλάβει σωστά ή λάθος; 🤔

**Comment by epidrome** [comment, marked as answer]
Συνολικά θα πρέπει να γίνουν δύο ασκήσεις μόνο από τον πίνακα της οπτικοποίησης
